cores/esp8266/uart.cpp

```diff
@@ -832,14 +826,18 @@ uart_set_tx(uart_t* uart, int tx_pin)
     case UART0:
         if(uart->tx_enabled)
         {
-            if (uart->tx_pin == 1 && tx_pin == 2)
+            if (uart->tx_pin == tx_pin)
+            {
+                return true;
+            }
```

Comment:
This is a fix, in the sense that this case was previously returning false, right ?